.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+repos:
+- repo: local
+  hooks:
+    - id: core
+      name: format core
+      language: system
+      entry: make -C libs/core format
+      files: ^libs/core/(langchain_core|tests)/
+      pass_filenames: false
+    - id: community
+      name: format community
+      language: system
+      entry: make -C libs/community format
+      files: ^libs/community/(langchain_community|tests)/
+      pass_filenames: false
+    - id: langchain
+      name: format langchain
+      language: system
+      entry: make -C libs/langchain format
+      files: ^libs/langchain/(langchain|tests)/
+      pass_filenames: false
+    - id: standard-tests
+      name: format standard-tests
+      language: system
+      entry: make -C libs/standard-tests format
+      files: ^libs/standard-tests/(langchain_standard_tests|tests)/
```

Comment:
might be worth adding the whole directory because changes to pyproject.toml/poetry.lock/makefile could make changes too, and this is what we detect in CI

Reply:
OK. I'll update

Reply:
Done.